src/abstract-api-adapter.coffee

```diff
@@ -0,0 +1,103 @@
+request = require 'request'
+try
+	{ Adapter } = require 'hubot'
+catch
+	prequire = require 'parent-require'
+	{ Adapter } = prequire 'hubot'
+
+class AbstractAPIAdapter extends Adapter
+	constructor: ->
+		super
+		@interval = 1000
```

Comment:
Ok, if this is intended I'm make this more explicit - e.g. a constructor argument that defaults to 1000.